 Index: jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStoryReporter.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStoryReporter.groovy	(date 1410374835000)
+++ jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStoryReporter.groovy	(revision )
@@ -588,7 +588,7 @@
             patterns.setProperty("beforeStory", "<div class=\"story-report\">\n");
 
             patterns.setProperty("description",
-                    "<div class='cm-jb-story-description-line'>{0}</div>\n<br>")
+                    "<div class=\"jb-story-report-description cm-jb-story-description-line\">{0}</div>\n<br>")
 
             patterns.setProperty("afterStory", "</div>\n");
             // TODO - canceled story?
Index: jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStepDocReporter.groovy
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStepDocReporter.groovy	(date 1410374835000)
+++ jbehave-for-jira/jbehave-for-jira-groovy-client/src/main/groovy/org/bitbucket/jbehaveforjira/groovyclient/GroovyStepDocReporter.groovy	(revision )
@@ -24,7 +24,7 @@
 import java.util.regex.Matcher
 import java.util.regex.Pattern
 /**
- * @author Maryna Stasyuk
+ * @author Maryna Pristrom
  */
 public class GroovyStepDocReporter implements StepdocReporter {
 
